src/3D/LandIsland.cpp

```diff
@@ -31,14 +30,6 @@ const float LandIsland::CellSize = 10.0f;
 LandIsland::LandIsland()
     : _blockIndexLookup {0}
 {
-	auto& filesystem = Game::instance()->GetFileSystem();
-	auto file = filesystem.Open(filesystem.TexturePath() / "smallbumpa.raw", FileMode::Read);
-	auto* smallbumpa = new uint8_t[file->Size()];
-	file->Read(smallbumpa, file->Size());
-
-	_textureSmallBump = std::make_unique<Texture2D>("LandIslandSmallBump");
-	_textureSmallBump->Create(256, 256, 1, Format::R8, Wrapping::Repeat, smallbumpa, static_cast<uint32_t>(file->Size()));
```

Comment:
Getting odd visual artifacts on my end. Different format perhaps?

![image](https://user-images.githubusercontent.com/32263167/165415773-27e7120f-3762-4a11-92a5-143371ab34e9.png)

Reply:
Logic within the loader for comparison:

```texture->Create(width, height, 1, format, graphics::Wrapping::ClampEdge, data.data(), static_cast<uint32_t>(data.size()));```